backend/persistent/datastores/Athena.js

```diff
@@ -0,0 +1,189 @@
+
+import {executeQuery} from './drivers/AWSAthenaDriver';
+
+import Logger from '../../logger';
+const SHOW_TABLES_QUERY = 'SHOW TABLES';
+const SHOW_SCHEMA_QUERY = 'SELECT table_name, column_name, data_type FROM '
+    + 'information_schema.columns WHERE table_schema ';
+const DEFAULT_QUERY_TIMEOUT = 2000;
+
+/*
+ * The connection function will validate the parameters and return the connection
+ * parameters
+ * @param {object} connection
+ * @param {string} connection.accessKey - AWS Access Key
+ * @param {string} connection.secretAccessKey - AWS Secret Key
+ * @param {string} connection.region - AWS Region
+ * @param {string} connection.database - Database name to connect to
+ * @param {string} connection.sqlStatement - SQL statement to execute
```

Comment:
no need for `sqlStatement` since `connect` only checks that it's possible to connect to the database.